### ISSUE TITLE
Preserve a directory structure in glob pattern download

### ISSUE THREAD

**Issue by danielc2013** [stelligent/crossing-go#12]
Background:
There is another ticket to allow for glob patterns. In this case, there should be a switch which determines if the directory is flattened, or preserved in the download

Depends On: https://github.com/stelligent/crossing-go/issues/11

ToDo:
Add a flag to crossing-go get which will determine if the directory structure is preserved on multi-file download. Something like --preserve, -p. The default behavior should also be defined.
Example:
`crossing-go get -p s3://my-big-bucket/**/*.go ./`
